edumanage/models.py

```diff
@@ -184,6 +264,24 @@ class Meta:
         verbose_name_plural = "Contacts"
 
 
+@python_2_unicode_compatible
+class Coordinates(models.Model):
```

Comment:
I think it would be helpful to document the units usedhere.

I assume it's degrees as float for lat/long - and meters for alt ... but I think it should better be documented.

Reply:
Right. I will try to add some `help_text` in some fields.

Reply:
Thanks, looks all good now.